go.mod

```diff
@@ -3,6 +3,9 @@ module github.com/nats-io/nats.go
 go 1.16
 
 require (
+	github.com/golang/protobuf v1.5.0
+	github.com/nats-io/nats-server/v2 v2.9.10 // indirect
```

Comment:
We keep testing dependencies outside of this go.mod by running tests with another modfile in this repo like this: 
```
go test -modfile=go_test.mod 
```